packages/hash/frontend/src/components/hooks/useCreateSubPage.ts

```diff
@@ -26,34 +29,33 @@ export const useCreateSubPage = (accountId: string) => {
     refetchQueries: ({ data }) => [
       {
         query: getAccountPagesTree,
-        variables: { accountId: data!.setParentPage.accountId },
+        variables: { ownedById: data!.setParentKnowledgePage.ownedById },
       },
     ],
   });
 
   const createSubPage = useCallback(
     async (parentPageEntityId: string, prevIndex: string | null) => {
       const response = await createPageFn({
-        variables: { accountId, properties: { title: "Untitled" } },
+        variables: { ownedById, properties: { title: "Untitled" } },
```

Comment:
Do we want to use `ownedById` here or the other one that we're using above? I'm a bit unclear about the differences between these

Reply:
I think it's the other way around, we can use `ownedById` directly in the above query because the parent page is always going to be in the same account as the sub-page

Reply:
Addressed in https://github.com/hashintel/hash/pull/1157/commits/71d4900c02a8e5e107f16f688ff18d02c07c0c96